lib/repo/git_repository.rb

```diff
@@ -2,6 +2,7 @@
 require 'gitolite'
 require 'digest/md5'
 require 'rubygems'
+require 'git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.